changelogs/fragments/6296-LanceNero-Terraform_statefile_check

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - LanceNero - Remove state file check condition and error block, because in the native implementation of terraform will not cause errors due to the non-existent file (https://github.com/ansible-collections/community.general/pull/6296)
```

Comment:
```suggestion
  - terraform - remove state file check condition and error block, because in the native implementation of terraform will not cause errors due to the non-existent file (https://github.com/ansible-collections/community.general/pull/6296).
```